script/Ciel/Day 14/14_01B_02_CIEL14B_4.txt

```diff
@@ -375,7 +375,8 @@ Although――I'm sure you don't have much will to fight anymore, do you?"
 [sha:c3c0d5f4b520974afd52ee00ce630c249be251e0]{
 -- Page 269, Offset 40222.
 -- 凄いな。萎えかけていた心臓に火が入った。
-Would you look at that.
+// Not sure about this one
+Would you look at that?
```

Comment:
I'm pro-no-question; may be worth refactoring if this format needs one

Reply:
Oh, there's another one in the next file lol. I guess add it and we can discuss Saturday.